packages/renderer/src/lib/container/ContainerColumnAge.svelte

```diff
@@ -0,0 +1,12 @@
+<script lang="ts">
```

Comment:
It'd still need a wrapper b/c this column shows text like 'Starting' too, but otherwise yes. There is no date exposed in ContainerInfoUI though so I didn't want to include here (and I think pods have the same issue).

Reply:
you could use rendermapping to convert it to a date object

Reply:
Not directly, since the column also uses the status. (I agree with direction, was saying more that it should be a future PR)